src/components/WorkPoolQueueStatusArray.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+  <div v-if="workPoolQueues.length < maxWorkQueues" class="work-pool-queue-status-array">
+    <WorkPoolQueueStatusIcon
+      v-for="workQueue in workPoolQueues"
+      :key="workQueue.id"
+      :work-queue-name="workQueue.name"
+      :work-pool-name="workPool.name"
+      class="work-pool-queue-status-badge__icon"
+    />
+  </div>
+  <span v-else>{{ workPoolQueues.length }}</span>
+</template>
+
+<script lang="ts" setup>
+  import { useSubscription } from '@prefecthq/vue-compositions'
+  import { computed } from 'vue'
+  import { WorkPoolQueueStatusIcon } from '@/components'
+  import { useWorkspaceApi } from '@/compositions'
+  import { WorkPool } from '@/models'
+
+  const props = defineProps<{
+    workPool: WorkPool,
+  }>()
+
+  const maxWorkQueues = 26
```

Comment:
what's the thought here? I assume showing 26 work queues doesn't look great so why not allow 27 or 37? I vote we let this look bad to encourage better patterns.

Reply:
So 25 would make 5 even rows of 5 in the actual implementation, this is worded poorly. I should make it 25 and use `<=`.

You can see from the percentiles Guidry pulled up:
<img width="240" alt="image" src="https://github.com/PrefectHQ/prefect-ui-library/assets/6776415/b73380e4-8ec9-4b0d-81a5-f7127cfd5e69">
Almost no one should encounter this, but for one workspace that has 454 work pools for some strange reason, hopefully this doesn't break is the real hope.. I could also bump this to 50 or something I suppose.

Reply:
Added a limit to 50 and a fallback.